Explorer/Assets/DCL/AvatarRendering/Wearables/Systems/LoadWearableAssetBundleManifestUtils.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Threading;
+using CommunicationData.URLHelpers;
+using Cysharp.Threading.Tasks;
+using DCL.Optimization.PerformanceBudgeting;
+using DCL.WebRequests;
+using ECS.StreamableLoading.Common.Components;
+using SceneRunner.Scene;
+using Utility;
+
+namespace DCL.AvatarRendering.Wearables.Systems
+{
+    public static class LoadWearableAssetBundleManifestUtils
+    {
+        private static readonly URLBuilder urlBuilder = new ();
+        private static readonly IWebRequestController webRequestController = IWebRequestController.DEFAULT;
```

Comment:
You should pass the web request controller as parameter instead of hardcoding a static instance

Reply:
Done!